oh_staff_ui/templates/oh_staff_ui/base.html

```diff
@@ -11,7 +11,14 @@
 </head>
 
 <body>
-    <p>Temporary: <a href="/item_search/">Search</a>&nbsp;<a href="/admin/">Admin</a>&nbsp;<a href="/logs/">Logs</a>&nbsp;<a href="/admin/logout/">Logout</a></p>
+    <ul class="navbar">
+        <li><a href="/item_search/">Search (by ARK or Title)</a></li>
+        <li><a href="/add_item/">Add Item</a></li>
```

Comment:
I added a link to `add_item`, which was the default URL route (which I also changed).